fix(vlist.sass): fix menu list items on IE11

Mikilll94 opened the pull request:
On IE11 text in menu list items was not vertically centered.

fix #10464

<!--
MAKE SURE TO READ THE CONTRIBUTING GUIDE BEFORE CREATING A PR
https://vuetifyjs.com/getting-started/contributing

Testing and markup sections can be removed for documentation changes
-->

<!-- Provide a general summary of your changes in the Title above -->
<!-- Keep the title short and descriptive, as it will be used as a commit message -->

<!-- We use conventional-changelog-angular for all commit structures -->
<!-- https://vuetifyjs.com/getting-started/contributing#commit-guidelines-w-commitizen -->


## Description
Fixes #10464

On IE11 text in menu list items is not vertically centered. This PR fixes this issue.
<!-- Describe your changes in detail -->
<!-- Note any issues that are resolved by this PR -->
<!-- e.g. resolves #4213 or fixes #2312 -->

## Motivation and Context
I wanted to fix the bug which was occurring only on IE11.
<!-- Why is this change required? What problem does it solve? -->
<!-- If it fixes an open issue, please link to the issue here. -->

## How Has This Been Tested?
Because this is a visual bug I have not created any unit tests.

I have tested this PR manually. First I have checked if on IE11 the behavior is correct. Then I have checked on Chrome and Firefox to see if everything is working as before. 
<!-- All PR's should implement unit tests if possible -->
<!-- Please describe how you tested your changes. -->
<!-- Have you created new tests or updated existing ones? -->
<!-- e.g. unit | visually | e2e | none -->

## Markup:
<!-- Information on how to setup your local development environment can be found here: -->
<!-- https://vuetifyjs.com/getting-started/contributing#setup-dev-environment -->

<!-- Paste markup for testing your change --->
<details>

```vue
<template>
  <v-container>
    <div class="text-center">
        <v-menu offset-y>
          <template v-slot:activator="{ on }">
            <v-btn
              color="primary"
              dark
              v-on="on"
            >
              Dropdown
            </v-btn>
          </template>
          <v-list>
            <v-list-item
              v-for="(item, index) in items"
              :key="index"
              @click="onItemClicked"
            >
              <v-list-item-title>{{ item.title }}</v-list-item-title>
            </v-list-item>
          </v-list>
        </v-menu>
      </div>
  </v-container>
</template>

<script>
export default {
  data() {
    return {
      items: [
        { title: "Click Me" },
        { title: "Click Me" },
        { title: "Click Me" },
        { title: "Click Me 2" }
      ]
    }
  },
  methods: {
    onItemClicked() {
    }
  }
}
</script>

```
</details>

## Types of changes
<!-- What types of changes does your code introduce? Put an `x` in all the boxes that apply: -->
- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Improvement/refactoring (non-breaking change that doesn't add any features but makes things better)

## Checklist:
<!-- Go over all the following points, and put an `x` in all the boxes that apply. -->
<!-- If you're unsure about any of these, don't hesitate to ask. We're here to help! -->
- [x] The PR title is no longer than 64 characters.
- [x] The PR is submitted to the correct branch (`master` for bug fixes and documentation updates, `dev` for new features and backwards compatible changes and `next` for non-backwards compatible changes).
- [x] My code follows the code style of this project.
- [ ] I've added relevant changes to the documentation (applies to new features and breaking changes in core library)

johnleider commented:
Was this a brute force fix or is there some flex knowledge you can bestow upon me!

Mikilll94 commented:
@johnleider 
Yeah, that was a brute force fix 😄 I removed it and added next commit with a real solution.

I have finally found the real cause of this bug. It is because on IE11 `align-items: center` is not working when `min-height` is set. This is described in details here:
https://github.com/philipwalton/flexbugs/issues/231

I have applied this workaround:
https://github.com/philipwalton/flexbugs/issues/231#issuecomment-362790042

Please check my changes again.

DRoet commented on packages/vuetify/src/components/VList/VListItem.sass:
```diff
@@ -40,6 +40,15 @@
+  min-height: inherit
+  font-size: 0
+
+
```
this might be cleaner  (adding to the existing v-list-item block)
```suggestion
  // Fix for IE11 where min-height does not work with
  // align-items: center in flex containers
  // https://github.com/philipwalton/flexbugs/issues/231
  &:after
    content: ''
    min-height: inherit
    font-size: 0
```

Mikilll94 replied:
Thanks. I did like you suggested.

johnleider commented on packages/vuetify/src/components/VList/VListItem.sass:
```diff
@@ -40,6 +40,15 @@
+  // Fix for IE11 where min-height does not work with
+  // align-items: center in flex containers
+  // https://github.com/philipwalton/flexbugs/issues/231
+  &:after
```
```suggestion
  &::after
```

Mikilll94 replied:
Thanks. I have applied your changes.
